test_runner/performance/test_perf_pgbench.py

```diff
@@ -232,3 +253,4 @@ def test_pgbench_remote_select_only(remote_compare: PgCompare, scale: int, durat
 @pytest.mark.remote_cluster
 def test_pgbench_remote_pgvector(remote_compare: PgCompare, duration: int):
     run_test_pgbench(remote_compare, 1, duration, PgBenchLoadType.PGVECTOR_HNSW)
+    run_test_pgbench(remote_compare, 1, duration, PgBenchLoadType.PGVECTOR_HALFVEC)
```

Comment:
I've noticed that this test got picked up also by the wrong job (`bench`) and it fails there because that job doesn't have correct env vars: https://github.com/neondatabase/neon/actions/runs/9493026851/job/26161115137

Let's move this test to a separate file and `--ignore` it here: https://github.com/neondatabase/neon/blob/a71f58e69c762e55f0fe9055a088f6232facbf28/.github/workflows/benchmarking.yml#L102

Reply:
good catch, fixed